web/source/osk/visualKeyboard.ts

```diff
@@ -222,8 +232,6 @@ namespace com.keyman.osk {
         ts.fontSize=spec['fontsize'];
       }
 
-      let activeKeyboard = com.keyman.singleton.core.activeKeyboard;
```

Comment:
👏 👏 👏  I take it this resolves a latent, undocumented bug in the OSK rendering, and makes it more functionally pure? 😄

Reply:
More functionally pure, to be sure.

And... I suppose it would solve a possible latent bug for documentation-keyboard renders, since the rendering keyboard generally isn't active.